server/src/fstar_handlers.ts

```diff
@@ -193,7 +32,10 @@ function handleIdeSymbol(textDocument: TextDocument, response: IdeSymbol, server
 }
 
 // If we get a proof state dump message, we store it in the proof state map
-function handleIdeProofState(textDocument: TextDocument, response: IdeProofState, server: Server) {
+//
+// TODO(klinvill): with my latest edits, this function isn't called anywhere.
+// Where should it be handled? server.ts? fstar_connection.ts?
```

Comment:
I think it should be called by whoever sends the full buffer query, i.e., `validateFStarDocument`.  (And the proof state messages should be included in the full buffer response stream.)

Unfortunately the full buffer protocol is a bit special.